### PR TITLE
Add Author Page and a new snppts

### PR DESCRIPTION
**Pull request by alexandresanlim**
**Add my first contrubuition! ;)**

- Subscription Plan 

### Add a new feature
- Author Page
I wold like get all projects from a author so i have create this:

When click detail project i have add a link in author name
image

When click this link i have redirect to all projects from author page
image

### CODE REVIEW

File: src/Snppts/Snppts.csproj [hunk 1]
```diff
@@ -23,7 +23,5 @@
     <Folder Include="Views\Shared\Components\PopularCategories\" />
     <Folder Include="Views\Shared\Components\AllCategories\" />
     <Folder Include="Views\Shared\Components\RandomSnippets\" />
-    <Folder Include="Snippets\" />
```
Comment (jfversluis) [inline src/Snppts/Snppts.csproj]: It's probably nothing and was done by Visual Studio automatically but could you undo the changes here please? Just to make sure we don't break anything.

Thanks!

**Review by alexandresanlim — COMMENTED** [review]
Revert remove lines

File: src/Snppts/Snppts.csproj (hunk 1 above)
Comment (alexandresanlim) [reply src/Snppts/Snppts.csproj]: It's ready ;)

File: src/Snppts/Snppts.csproj [hunk 2]
```diff
@@ -9,11 +9,11 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.4.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.4" />
     <PackageReference Include="X.PagedList.Mvc.Core" Version="7.6.0" />
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="Views\Snippets\" />
```
Comment (jfversluis) [inline src/Snppts/Snppts.csproj]: Still a line is being removed. And I see you added a new feature now as well? :)

Comment (alexandresanlim) [reply src/Snppts/Snppts.csproj]: I dont know why the visual studio remove this line but i reverted again this change.
Yea man! I add author page, i have change the description to info this change.
I should create a new branch? probably yes, in next time i will do this. ;)

tks!

**Review by jfversluis — APPROVED** [review]

File: src/Snppts/Snppts.csproj (hunk 2 above)
Comment (jfversluis) [reply src/Snppts/Snppts.csproj]: Awesome! Yeah! Love new features, but indeed next time maybe a separate PR for that. But sill much appreciated!
